File: DefaultCodeInputOutput/1.js
    const fs = require('fs');

    function twoSum(nums, target) {
    //write here your code
  
    }
    
    if (process.argv.length !== 3) {
        console.log("Usage: node script.js '{\"nums\": [...], \"target\": ...}'");
        process.exit(1);
    }
    
    const inputJson = process.argv[2];
    try {
        const inputData = JSON.parse(inputJson);
        const nums = inputData.nums;
        const target = inputData.target;
        console.log(twoSum(nums, target));
    } catch (e) {
        console.log("Invalid input format. Please provide a JSON string with 'nums' and 'target'.");
    }
    

File: DefaultCodeInputOutput/1.py
import sys
import json
from modeInclude import *

#def two_sum(nums, target):
    #write here your code
    

if __name__ == "__main__":
    if len(sys.argv) != 2:
        print("Usage: python script.py '{\"nums\": [...], \"target\": ...}'")
        sys.exit(1)

    input_json = sys.argv[1]
    try:
        input_data = json.loads(input_json)
        nums = input_data["nums"]
        target = input_data["target"]
        print(two_sum(nums, target))
    except (json.JSONDecodeError, KeyError):
        print("Invalid input format. Please provide a JSON string with 'nums' and 'target'.")


